common/service/dynamicconfig/constants.go

```diff
@@ -339,6 +341,11 @@ const (
 	FrontendThrottledLogRPS
 	// EnableClientVersionCheck enables client version check for frontend
 	EnableClientVersionCheck
+	// EnableRawHistoryQuery is whether to enable raw history retrieving
+	EnableRawHistoryQuery
+	// EnableRawHistoryQuery is whether to enable raw history retrieving by domain
+	EnableRawHistoryQueryByDomain
```

Comment:
Why do we need two config knobs ? If you create one byDomain you can always use a default value that applies to all domains.

Reply:
> Why do we need two config knobs ? If you create one byDomain you can always use a default value that applies to all domains.

I saw consistent query is set up with two knobs for some reason. Just made it work in the same way. Anyway, I have removed the extra one.